Gemfile

```diff
@@ -8,6 +8,7 @@ if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.2.2')
 end
 
 gem 'ffi', '>= 1.9.14'
+gem 'htmlentities'
```

Comment:
This should go in the gemspec since the InSpec core (and therefore users who install the gem) is relying on it.